use-timescale/continuous-aggregates/about-continuous-aggregates.md

```diff
@@ -55,13 +55,51 @@ these restrictions:
     clause does not matter.
 *   You must use an `INNER JOIN`, no other join type is supported.
 *   The `JOIN` conditions can only be equality conditions.
-*   The hypertable on the `JOIN` condition must be a hypertable, and not a continuous
-     aggregate. Additionally, you can't use joins in hierarchical continuous aggregates.
+*   The hypertable on the `JOIN` condition can be a hypertable, and not a continuous
+    aggregate. Additionally, you can't use joins in hierarchical continuous aggregates.
 *   Changes to the hypertable are tracked, and are updated in the continuous aggregate
      when it is refreshed. Changes to the standard PostgreSQL table are not tracked.
 *   The `USING` clause is supported in joins for PostgreSQL&nbsp;13 or later. In
     PostgreSQL&nbsp;12 only joins with a condition in the `WHERE` clause or
     `JOIN` clause can be used.
+*   Conditions other than JOIN condition can be added to the query when using
+    ON/USING clause to specify the join conditions and other conditions can be added to
+    the WHERE clause. There is no restrictions on the number of conditions in this
+    WHERE clause but only ONE condition can be given in ON/USING clause which has to be
+    EQUALITY condition.
+    Note that, if ON/USING clause is not used and all the conditions
+    including the JOIN condition are given in WHERE clause then query will err.
+    Also, if ON/USING clause is not used then only ONE condition which will be considered as
+    JOIN condition can be given in WHERE clause.
+
+    Concretely, with refrence to JOIN conditions queries that will work are,
+    CREATE MATERIALIZED VIEW my_view WITH (timescaledb.continuous) AS
+    SELECT ...
+    FROM table_1 t1
+    JOIN table_2 t2 ON t1.t2_id = t2.id
+
+    CREATE MATERIALIZED VIEW my_view WITH (timescaledb.continuous) AS
+    SELECT ...
+    FROM table_1 t1
+    JOIN table_2 t2 ON t1.t2_id = t2.id
+    WHERE t1.id IN (1, 2, 3, 4)
+
+    CREATE MATERIALIZED VIEW my_view WITH (timescaledb.continuous) AS
+    SELECT ...
+    FROM table_1 t1, table_2 t2
+    WHERE t1.t2_id = t2.id
+
+    The variations that will NOT work are,
+    CREATE MATERIALIZED VIEW my_view WITH (timescaledb.continuous) AS
+    SELECT ...
+    FROM table_1 t1
+    JOIN table_2 t2 ON t1.t2_id = t2.id AND t1.t2_id_2 = t2.id
+
+    CREATE MATERIALIZED VIEW my_view WITH (timescaledb.continuous) AS
+    SELECT ...
+    FROM table_1 t1, table_2 t2
+    WHERE t1.t2_id = t2.id
+    AND t1.id IN (1, 2, 3, 4)
```

Comment:
If I try to do this in the GH web UI, it'll break everything, but these need to be in individual code blocks, with three backticks, and `sql` declared as the language.